x-pack/plugins/security_solution/public/timelines/components/timeline/selectable_timeline/index.tsx

```diff
@@ -276,13 +271,16 @@ const SelectableTimelineComponent: React.FC<SelectableTimelineProps> = ({
         listProps={{
           rowHeight: TIMELINE_ITEM_HEIGHT,
           showIcons: false,
-          virtualizedProps: ({
-            onScroll: handleOnScroll.bind(
-              null,
-              timelines.filter((t) => !hideUntitled || t.title !== '').length,
-              timelineCount
-            ),
-          } as unknown) as ListProps,
+          windowProps: {
+            onScroll: ({ scrollOffset }) =>
+              handleOnScroll(
+                timelines.filter((t) => !hideUntitled || t.title !== '').length,
+                timelineCount,
+                scrollOffset
+              ),
+            outerRef: selectableListOuterRef,
+            innerRef: selectableListInnerRef,
+          },
```

Comment:
The underlying virtualization library changed and the `onScroll` API along with it. All the pieces were still available to perform `handleOnScroll` as expected, but it required a "roll your own" approach.